src/dream-team.js

```diff
@@ -1,6 +1,14 @@
 const CustomError = require("../extensions/custom-error");
 
-module.exports = function createDreamTeam(/* members */) {
-  throw new CustomError('Not implemented');
-  // remove line with error and write your code here
+module.exports = function createDreamTeam(teamMembersArray) {
+  let teamName = '';
+  let teamCheck = '';
+  if(!Array.isArray(teamMembersArray)){
+    return false;
+  }
+  teamCheck = teamMembersArray.filter(name => typeof(name) == "string" );
+  for(let i=0; i<teamCheck.length; i++){
+    teamName += teamCheck[i].replace(/\s/g, '').charAt(0).toUpperCase();
+  }
+  return teamName.split('').sort().join('');
```

Comment:
Again, try to use reduce to not create useless variables. An you can skip filter step with reduce, because you can do this right iside reducer. Also you can use es6 spread operator. Smth like this:
return teamMembersArray.reduce((acc, name) => {
typeof name === 'string' ? [...acc, name.trim().charAt(0).toUpperCase() ] : acc,
[])
.sort()
.join('');

Reply:
Fixed, ready to review